docs/tutorials/zero-to-hero.md

```diff
@@ -41,130 +32,35 @@ Let's get started!
 >
 > You can interact with example apps online, or explore the code by opening it in gitpod (online IDE). To do that, click on the example you want to see, and then click "Open in gitpod". Gitpod online IDE will open with the project loaded.
 
-All our sample projects work out of the box. Pick any assemblyscript project to proceed (for example, guest book). The project file structure is explained in more detail [here](/docs/quick-start/development-overview)
+All our sample projects work out of the box. Pick the  (for example, guest book). The project file structure is explained in more detail [here](/docs/quick-start/development-overview)
 
 Let's look over the directory and introduce you to the *main files* you'll be interacting with during this tutorial.  These are the same files you will almost always work with when prototyping ideas using gitpod.  For deeper work on your local machine you can also download the project but please do not do that right now.
 
 - `assembly/main.ts` - This is where the smart contract code goes.
 - `assembly/model.ts` - Define the types you want to use in your contract here.
-- `src/index.html` - Basic layout for your front end
-- `src/main.js` - Wire the logic for your app here
-- `src/test.js` - For you to write tests
+- `src/App.js` - Basic layout for your front end (React app)
+- `src/main.js` - Wire up the backend logic for your app here
+- `src/test.js` - Tests for the backend logic.
```

Comment:
let's add `src/index.html` (see comment below)

Reply:
personally not a fan of adding React to any of our tutorials bc it raises the bar. no need to change this, just my 2 cents

Reply:
oh this is out of date. this is not react. fixed